### PR TITLE
storage: correct struct fields in volume plugins

### PR DESCRIPTION
**Pull request by humblec**
This commit corrects struct field names in the godoc for various
storage volume plugins volume sources and persistent volume sources.

Additional Ref# #105963 (comment)

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

/kind cleanup

```release-note
NONE

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -718,15 +718,15 @@ type EmptyDirVolumeSource struct {
 // Represents a Glusterfs mount that lasts the lifetime of a pod.
 // Glusterfs volumes do not support ownership management or SELinux relabeling.
 type GlusterfsVolumeSource struct {
-	// EndpointsName is the endpoint name that details Glusterfs topology.
+	// endpointsName is the endpoint name that details Glusterfs topology.
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:721]:
```suggestion
	// endpoints is the endpoint name that details Glusterfs topology.
```

File: staging/src/k8s.io/api/core/v1/types.go [hunk 2]
```diff
@@ -1247,42 +1247,42 @@ type NFSVolumeSource struct {
...
-	// Target iSCSI Qualified Name.
+	// iqn is the target iSCSI Qualified Name.
 	IQN string `json:"iqn" protobuf:"bytes,2,opt,name=iqn"`
-	// iSCSI Target Lun number.
+	// lun represents iSCSI Target Lun number.
 	Lun int32 `json:"lun" protobuf:"varint,3,opt,name=lun"`
 	// iSCSI Interface Name that uses an iSCSI transport.
 	// Defaults to 'default' (tcp).
 	// +optional
 	ISCSIInterface string `json:"iscsiInterface,omitempty" protobuf:"bytes,4,opt,name=iscsiInterface"`
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:1260]: update doc to `iscsiInterface ...`

File: staging/src/k8s.io/api/core/v1/types.go [hunk 3]
```diff
@@ -1339,78 +1339,78 @@ type ISCSIPersistentVolumeSource struct {
...
 	WWIDs []string `json:"wwids,omitempty" protobuf:"bytes,5,rep,name=wwids"`
 }
 
 // AzureFile represents an Azure File Service mount on the host and bind mount to the pod.
 type AzureFileVolumeSource struct {
-	// the name of secret that contains Azure Storage Account Name and Key
+	// secretName is the  name of secret that contains Azure Storage Account Name and Key
 	SecretName string `json:"secretName" protobuf:"bytes,1,opt,name=secretName"`
-	// Share Name
+	// sharename is the azure share Name
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:1368]:
```suggestion
	// shareName is the azure share Name
```

File: staging/src/k8s.io/api/core/v1/types.go [hunk 4]
```diff
@@ -1339,78 +1339,78 @@ type ISCSIPersistentVolumeSource struct {
...
 	ShareName string `json:"shareName" protobuf:"bytes,2,opt,name=shareName"`
-	// Defaults to false (read/write). ReadOnly here will force
+	// readONly defaults to false (read/write). ReadOnly here will force
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:1370]:
```suggestion
	// readOnly defaults to false (read/write). ReadOnly here will force
```

File: staging/src/k8s.io/api/core/v1/types.go [hunk 5]
```diff
@@ -1606,7 +1606,7 @@ type StorageOSPersistentVolumeSource struct {
 // ConfigMap volumes support ownership management and SELinux relabeling.
 type ConfigMapVolumeSource struct {
 	LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
-	// If unspecified, each key-value pair in the Data field of the referenced
+	// Items if unspecified, each key-value pair in the Data field of the referenced
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:1609]:
```suggestion
	// items if unspecified, each key-value pair in the Data field of the referenced
```

File: staging/src/k8s.io/api/core/v1/types.go [hunk 6]
```diff
@@ -1699,16 +1699,16 @@ type ProjectedVolumeSource struct {
...
-	// information about the secret data to project
+	// secret information about the secret data to project
 	// +optional
 	Secret *SecretProjection `json:"secret,omitempty" protobuf:"bytes,1,opt,name=secret"`
-	// information about the downwardAPI data to project
+	// downwardAPI information about the downwardAPI data to project
 	// +optional
 	DownwardAPI *DownwardAPIProjection `json:"downwardAPI,omitempty" protobuf:"bytes,2,opt,name=downwardAPI"`
-	// information about the configMap data to project
+	// configmap information about the configMap data to project
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:1708]:
```suggestion
	// configMap information about the configMap data to project
```

**Review by liggitt — COMMENTED** [review]
noted some missing/typoed field names. also do a sweep to make sure what follows the field name is a coherent phrase

**Comment by humblec** [comment]
> noted some missing/typoed field names. also do a sweep to make sure what follows the field name is a coherent phrase

Sharp as always :) , sure, addressed those comments .. Thanks @liggitt, ptal !

**Pull request merged**